README.md

```diff
@@ -60,28 +61,34 @@ Inspired by all the other [awesome lists](https://github.com/sindresorhus/awesom
 ## Communication
 - [Yale Program on Climate Change Communication](https://climatecommunication.yale.edu) - Scientific research on public climate change knowledge, attitudes, policy preferences, and behavior, and the underlying psychological, cultural, and political factors that influence them.
 - [The Psychology of Climate Change Communication](http://guide.cred.columbia.edu/pdfs/CREDguide_full-res.pdf) - A Guide for Scientists, Journalists, Educators, Political Aides, and the Interested Public.
+- [Skeptical Science](https://skepticalscience.com/) - Non-profit science education organization with the goal of raising public understanding of climate change. Specifically addresses common myths and talking points about climate change.
```

Comment:
I wonder if this should be in a new section - for example ##Science?

The idea for the Communication section was to list resources on how the science and impacts are communicated and felt rather than the science itself.